src/parser/validate_args.c

```diff
@@ -11,21 +11,24 @@ int	args_valid(int argc, char **argv)
 
 	if (argc != 2)
```

Comment:
maybe add a message or something here if arg_count is wrong
i know it was imlpemented in an earlier request, but still could be nice :)